Directory.Build.targets

```diff
@@ -79,10 +79,13 @@
     <Content Remove="@(None->WithMetadataValue('Pack','true'))" />
   </ItemGroup>
 
-  <!-- Until we get a new enough dotnet -->
-  <ItemGroup Condition="'$(_MauiTargetPlatformIsWindows)' == 'True'">
-    <FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.19041.31" TargetingPackVersion="10.0.19041.31" />
+  <!-- Until we get net9 rtm -->
```

Comment:
So since this is not in the internal rc2 we don't need this, and this can be deleted even when updating to rtm right @Redth  ?